lib/ProductOpener/Ingredients.pm

```diff
@@ -1626,10 +1626,20 @@ sub parse_ingredients_text ($product_ref) {
 
 					# sel marin (France, Italie)
 					# -> if we have origins, put "origins:" before
-					if (    ($between =~ $separators)
-						and (exists_taxonomy_tag("origins", canonicalize_taxonomy_tag($ingredients_lc, "origins", $`))))
+					if (
+						(
+							($between =~ /$separators| and /)
```

Comment:
Maybe this instead?
It's for things like "Sel marin (France et Italie)"? Maybe add a test for it.

```suggestion
							($between =~ /$separators|$and/)
```

Reply:
Not sure... Japenese for " and " would be "と" (although not in Ingredients.pm yet in %and) and it has to be without space. Then if you have japanese-words-with-と separated by と, that will split the japanese-words-with-と.

Eventually, we could try this:
```suggestion
							($between =~ /$separators| and |$and/)
```
That would use " and " for Japanese and $and for other defined languages.

What do you think?

Reply:
What I don't understand is why you changed "($between =~ $separators)" to "($between =~ /$separators| and /)". Why add "| and "? It would help parse something like "Tomatoes (France and Italy), in which case we could add a test for it. But it would not match "Tomates (France et Italie).

That's why I suggested ($between =~ /$separators|$and/)

Note that we don't have と in %and.

And we have "my $and = $and{$ingredients_lc} || " and ";" meaning that in Japanese, $and will be equal to " and ", so it wont match something with と in it.

To make sure it works, could you add test cases like "Tomatoes (France and Italy)", "Tomates (France and Italy)", and a Japanese one with a と in it which could be "やきとり(ときがわ町])" for instance.

Reply:
I mixed up %and and $and... :-(